test_unstructured_ingest/check-diff-evaluation-metrics.sh

```diff
@@ -13,43 +13,50 @@ set -e
 SCRIPT_DIR=$(dirname "$(realpath "$0")")
 OVERWRITE_FIXTURES=${OVERWRITE_FIXTURES:-false}
 TMP_DIRECTORY_CLEANUP=${TMP_DIRECTORY_CLEANUP:-true}
+OUTPUT_FOLDER_NAME=$1
 OUTPUT_ROOT=${OUTPUT_ROOT:-$SCRIPT_DIR}
-TMP_METRICS_LATEST_RUN_DIR=$OUTPUT_ROOT/metrics-tmp
-METRICS_DIR=$OUTPUT_ROOT/metrics
+TMP_METRICS_LATEST_RUN_DIR=$OUTPUT_ROOT/metrics-tmp/$OUTPUT_FOLDER_NAME
+METRICS_DIR=$OUTPUT_ROOT/metrics/$OUTPUT_FOLDER_NAME
 
 # shellcheck disable=SC1091
 source "$SCRIPT_DIR"/cleanup.sh
 
 function cleanup() {
+  if [ "$TMP_DIRECTORY_CLEANUP" == "true" ]; then
     cleanup_dir "$TMP_METRICS_LATEST_RUN_DIR"
+  else
+    echo "skipping tmp directory cleanup"
+  fi
 }
 
 trap cleanup EXIT
 
-echo "running check diff"
-echo "$TMP_METRICS_LATEST_RUN_DIR"
-echo "$METRICS_DIR"
-
 # to update ingest test fixtures, run scripts/ingest-test-fixtures-update.sh on x86_64
 if [ "$OVERWRITE_FIXTURES" != "false" ]; then
+    # remove folder if it exists
+    if [ -d "$METRICS_DIR" ]; then
+        rm -rf "$METRICS_DIR"
+        # find "$METRICS_DIR" -maxdepth 1 -type f ! -name "metrics-json-manifest.txt" -exec rm -rf {} +
+    fi
     # force copy (overwrite) files from metrics-tmp (new eval metrics) to metrics (old eval metrics)
-    cp -f "$TMP_METRICS_LATEST_RUN_DIR"/* "$METRICS_DIR"
-# elif ! diff -ru "$METRICS_DIR" "$TMP_METRICS_LATEST_RUN_DIR" ; then
-#     "$SCRIPT_DIR"/clean-permissions-files.sh "$TMP_METRICS_LATEST_RUN_DIR"
-#     diff -r "$METRICS_DIR" "$TMP_METRICS_LATEST_RUN_DIR"> outputdiff.txt
-#     cat outputdiff.txt
-#     diffstat -c outputdiff.txt
-#     echo
-#     echo "There are differences from the previously checked-in structured outputs."
-#     echo
-#     echo "If these differences are acceptable, overwrite by the fixtures by setting the env var:"
-#     echo
-#     echo "  export OVERWRITE_FIXTURES=true"
-#     echo
-#     echo "and then rerun this script."
-#     echo
-#     echo "NOTE: You'll likely just want to run scripts/ingest-test-fixtures-update.sh on x86_64 hardware"
-#     echo "to update fixtures for CI."
-#     echo
-#     exit 1
+    mkdir -p "$METRICS_DIR"
+    cp -rf "$TMP_METRICS_LATEST_RUN_DIR" "$OUTPUT_ROOT/metrics"
+elif ! diff -ru "$METRICS_DIR" "$TMP_METRICS_LATEST_RUN_DIR" ; then
+    "$SCRIPT_DIR"/clean-permissions-files.sh "$TMP_METRICS_LATEST_RUN_DIR"
+    diff -r "$METRICS_DIR" "$TMP_METRICS_LATEST_RUN_DIR"> metricsdiff.txt
+    cat metricsdiff.txt
+    diffstat -c metricsdiff.txt
+    echo
+    echo "There are differences from the previously checked-in structured outputs."
+    echo
+    echo "If these differences are acceptable, overwrite by the fixtures by setting the env var:"
+    echo
+    echo "  export OVERWRITE_FIXTURES=true"
+    echo
+    echo "and then rerun this script."
+    echo
+    echo "NOTE: You'll likely just want to run scripts/ingest-test-fixtures-update.sh on x86_64 hardware"
+    echo "to update fixtures for CI."
+    echo
+    exit 1
```

Comment:
since there are other updates in this PR, let's still comment out the failure case so that can be an indpendent PR.